packages/opensrp-plans/src/helpers/dataLoaders.ts

```diff
@@ -68,3 +85,325 @@ export async function loadSinglePlan(
       throw err;
     });
 }
+
+/** get all the assignments for plan
+ *
+ * @param {string} baseURL -  base url of api
+ * @param {string} planId - id of the product to be fetched
+ * @param {OpenSRPService} service - the opensrp service
+ * @param {fetchAssignments} actionCreator - Action creator; creates actions that adds plans to the store
+ * @returns {Promise<void>}
+ */
+export async function loadAssignments(
+  baseURL: string,
+  planId: string,
+  service: typeof OpenSRPService = OpenSRPService,
+  actionCreator: typeof fetchAssignments = fetchAssignments
+) {
+  // get all assignments
+  const serve = new service(OPENSRP_GET_ASSIGNMENTS_ENDPOINT, baseURL);
+  return serve
+    .list({ plan: planId })
+    .then((response: RawAssignment[] | null) => {
+      if (response) {
+        const receivedAssignments: Assignment[] = processRawAssignments(response);
+        // save assignments to store
+        actionCreator(receivedAssignments);
+      } else {
+        throw new Error(COULD_NOT_LOAD_ASSIGNMENTS);
+      }
+    })
+    .catch((e) => {
+      throw e;
+    });
+}
+
+/**
+ * load organizations for plan
+ *
+ * @param {string} baseURL -  base url of api
+ * @param {OpenSRPService} service - the opensrp service
+ * @param {fetchOrganizations} actionCreator - Action creator; creates actions that adds plans to the store
+ * @returns {Promise<void>}
+ */
+export async function loadOrganizations(
+  baseURL: string,
+  service: typeof OpenSRPService = OpenSRPService,
+  actionCreator: typeof fetchOrganizations = fetchOrganizations
+) {
+  // fetch all organizations
+  const serve = new service(OPENSRP_ORGANIZATION_ENDPOINT, baseURL);
```

Comment:
Also `OPENSRP_ORGANIZATION_ENDPOINT`

Reply:
I have been thinking about this, And I think they more behave like constants do, in the sense that if say one changes, it should & will uniformly do so across the package(i.e. a single change in the constants file). Also, I do not expect the endpoints to have different string values across function calls, I also think the string values of the endpoints are highly stable in that they will not change from the api side which makes me think there isn't much use in having them as configurable.
Do you still think I should still have them as arguments, am not entirely opposed to it

Reply:
My concern was in case the endpoint changes the package has to be published again. 
Yes the API is stable but was of the opinion of passing them as params with default values.

Reply:
cool, I will make the change.

Reply:
will move this into an issue: https://github.com/OpenSRP/web/issues/301